### PR TITLE
Simplest support of Windows with Docker

### LINKED ISSUE #1: Windows support
Perhaps mention in the README that because this depends on vty it won't work on Windows, see https://github.com/jtdaugherty/vty/issues/119.

### PR DESCRIPTION
**Pull request by igrep**
May fix #1 
But there're still problems:
- Requires `cygpath` (tool to convert Windows path into the POSIX one)
- Can't handle multiple arguments and options. (Is that really necessary for this tool?)

Tried with: [Docker Toolbox v17.04.0ce](https://github.com/docker/toolbox/releases/tag/v17.04.0-ce)

### CODE REVIEW

**Comment by maoe** [comment]
Thanks for your contribution! Could you extend the README to show how to use it?

File: README.md [hunk 1]
```diff
@@ -17,12 +17,31 @@ It has three display modes:
...
+    - Bundled with [Cygwin](https://www.cygwin.com/), [MSYS2](http://www.msys2.org/) etc.
+
+#### How to install and launch
+
+```bash
+$ git clone git@github.com:maoe/viewprof.git
+$ cd viewprof/docker
+$ docker-machine start
+$ docker build -t viewprof .
+$ .\viewprof.bat path/to/file.prof
```
Comment (maoe) [inline README.md]: Does Windows accept forward slash delimiters?

File: README.md [hunk 2]
```diff
@@ -17,12 +17,31 @@ It has three display modes:
...
+#### Prerequisites
+
+- [Docker for Windows](https://docs.docker.com/docker-for-windows/install/) or [Docker Toolbox](https://docs.docker.com/toolbox/overview/).
+    - Currently tried with: [Docker Toolbox v17.04.0ce](https://github.com/docker/toolbox/releases/tag/v17.04.0-ce) on Windows 10 Home.
+- [cygpath](https://cygwin.com/cygwin-ug-net/cygpath.html)
+    - Bundled with [Cygwin](https://www.cygwin.com/), [MSYS2](http://www.msys2.org/) etc.
+
+#### How to install and launch
+
+```bash
```
Comment (maoe) [inline README.md]: Should we use `bat` or something instead? Or is it actually bash?

File: README.md [hunk 3]
```diff
@@ -17,12 +17,31 @@ It has three display modes:
...
+$ git clone git@github.com:maoe/viewprof.git
```
Comment (maoe) [inline README.md]: Please drop the $-signs to make the instructions copy-paste friendly.

File: README.md [hunk 4]
```diff
@@ -17,12 +17,31 @@ It has three display modes:
 
 ## Installation
 
-Note: Currently viewprof doesn't support Windows because the underlying library (vty) doesn't support it yet. See [#1](https://github.com/maoe/viewprof/issues/1).
+For Windows users, see below.
```
Comment (maoe) [inline README.md]: Let's drop this sentence. The following instruction is short.

**Review by maoe — CHANGES_REQUESTED** [review]
Thanks! I've commented a few places.

File: README.md (hunk 1 above)
Comment (igrep) [reply README.md]: Yes, actually whichever will do.

Comment (maoe) [reply README.md]: That's good to know.

**Review by maoe — APPROVED** [review]
LGTM, thanks!

**Comment by maoe** [comment]
Squashed last three commits and merged into feature/docker.

**Pull request closed without merging**